PerformanceTesting/ecl/09aa_indexread123.ecl

```diff
@@ -18,4 +18,12 @@ ds := files.manyIndex123(
              
 cnt := COUNT(NOFOLD(ds));
 
-OUTPUT(cnt = 256 * 256 * 2);
+indexRecords := COUNT(files.manyIndex123);
+
+expected := MAP(
```

Comment:
this allows the test to work with significantly lower values for config.memoryPerSlave